s3fs/core.py

```diff
@@ -71,6 +71,59 @@ def split_path(path):
              'authenticated-read'}
 
 
+def title_case(string):
```

Comment:
It would be nice to put this and the helpers below into util instead, a little cleaner.